src/Main/Main.cc

```diff
@@ -46,14 +46,13 @@ int main(int argc, char* argv[]) {
         sigaction(SIGINT, &sig_handler, nullptr);
 
         // Main
-        carta::ProgramSettings settings(argc, argv);
+        auto settings = ProgramSettings::GetInstance() = std::move(carta::ProgramSettings(argc, argv));
```

Comment:
This is a little awkward. I would suggest adding an `Initialise` static method to `ProgramSettings` which initialises the instance internally, and calling it here (something like: `ProgramSettings::Initialise(argc, argv)`).